CHANGELOG.md

```diff
@@ -3,6 +3,38 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.17.23
+
+_Feb 9, 2023_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```

Comment:
```suggestion
- 🌍 Improve Brazilian Portuguese (pt-BR) locale
- 🐞 Bugfixes
```

Reply:
Maybe one more place 
```suggestion
- 🌍 Improve Brazilian Portuguese (pt-BR) locale
- 🎉 Add banner and callouts to inform about MUI X v6 beta
- 🐞 Bugfixes
```
Maybe yet another opportunity/place to increase visibility? 🙈